bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -313,6 +315,36 @@
    @XmlElement
    private String                timeZoneId;
 
+   /**
+    * An array of {@link TourTimerPause}
+    * A timer pause is a device event triggered by the user.
+    * TODO FB rename to ?
+    * tourDevicePauses
+    * tourUserPauses
+    * .....
+    */
+   //TODO FB Can't i map it just with the tourid ???
+   @OneToMany(fetch = FetchType.EAGER, cascade = ALL, mappedBy = "tourData")
+   @Cascade(org.hibernate.annotations.CascadeType.DELETE_ORPHAN)
+   @XmlElementWrapper(name = "TourTimerPauses")
+   @XmlElement(name = "TourTimerPause")
+   private List<TourTimerPause>             tourTimerPauses                         = new ArrayList<>();
```

Comment:
The EJB list field `tourTimerPauses` should be moved to the other EJB list fields (about line 888) that all of them are together

Reply:
Fixed